src/components/CategoriesPills.js

```diff
@@ -0,0 +1,144 @@
+// @flow
+import React from "react";
+import { View, StyleSheet, ScrollView } from "react-native";
+import LinearGradient from "react-native-linear-gradient";
+import type { StyleObj } from "react-native/Libraries/StyleSheet/StyleSheetTypes";
+import Text from "./Text";
+import text from "../constants/text";
+import eventCategories from "../constants/event-categories";
+import {
+  blackColor,
+  whiteColor,
+  coralColor,
+  darkBlueGreyTwoColor,
+  eucalyptusGreenColor
+} from "../constants/colors";
+
+const locale = "en-GB";
```

Comment:
Localization has been considered in the components, but needs further work to fully integrate.

Reply:
I guess @jonyardley did that already https://github.com/redbadger/pride-london-app/pull/76